README.md

```diff
@@ -107,7 +107,7 @@ Community Maintained Cars are not confirmed by comma.ai to meet our [safety mode
 In Progress Cars
 ------
 - All TSS-P Toyota with Steering Assist.
-  - 'Full Speed Range Dynamic Radar Cruise Control' is required to enable stop-and-go. Only the Prius, Camry and C-HR have this option.
+  - 'Full Speed Range Dynamic Radar Cruise Control' is required to enable stop-and-go. Only the Prius, Camry and C-HR have this option.  OpenPilot also adds stop-an-go functionality for Highlander and Highlander Hybrid.
```

Comment:
I would remove this change. But yes, later on we'll add something to indicate which cars gain stop and go.

Reply:
Fixed